test/setup/iam.tf

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  log_export_required_roles = [
+    # Needed for the Pubsub submodule to create a service account for the
+    # subscription it creates
+    "roles/iam.serviceAccountAdmin",
+
+    # Needed for the cloud storage submodule to create/delete a bucket
+    "roles/storage.admin",
+
+    # Needed for the pubsub submodule to create/delete a pubsub topic
+    "roles/pubsub.admin",
+
+    # Needed for the bigquery submodule to create/delete a bigquery dataset
+    "roles/bigquery.dataOwner",
+
+    # Needed for the root module to activate APIs
+    "roles/serviceusage.serviceUsageAdmin",
+
+    # Needed for the Pubsub submodule to assign roles/bigquery.dataEditor to
+    # the service account it creates
+    "roles/resourcemanager.projectIamAdmin",
+
+    # Required to create log sinks from the project level
+    "roles/logging.configWriter",
+  ]
+
+  log_export_billing_account_roles = [
+    # Required to associate billing accounts to new projects
```

Comment:
Are you sure we need this? The tests themselves shouldn't be creating any new projects (the project creation happens in the prepare step).

Reply:
The prepare [creates project](https://github.com/ivankorn/terraform-google-log-export/blob/29/test/setup/main.tf#L17-L40) while the computed values tests is [doing association](https://github.com/ivankorn/terraform-google-log-export/blob/29/test/fixtures/computed_values/main.tf#L23-L28) to billing account